### PR TITLE
DE-237/cicd-documentation

### PR DESCRIPTION
**Pull request by valoumiaou**
# DE PR

## Describe your changes

Please include a summary of the changes:

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] My code passes CI/CD tests
- [ ] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: .github/workflows/CD_workflow.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: "Deploy Composer"
+name: "Deployement workflow"
```
Comment (lmontier-pass) [inline .github/workflows/CD_workflow.yml:1]: Trop bien ce renaming ! 
Est-ce qu'on appellerait pas ça CD workflow pour être iso avec le nom du fichier (et retrouver plus facilement qui est quoi)?

File: .github/workflows/CI_workflow.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: "Base Workflow"
+name: "Integration Workflow"
```
Comment (lmontier-pass) [inline .github/workflows/CI_workflow.yml:1]: CI workflow du coup

File: .github/workflows/CI_workflow.yml [hunk 3]
```diff
@@ -20,23 +20,23 @@ jobs:
...
       GCP_EHP_SERVICE_ACCOUNT: ${{ secrets.GCP_EHP_SERVICE_ACCOUNT }}
-   compile-dbt-on-stg:
+   dbt-integration-stg:
     needs: find-dbt-changes
     if: github.base_ref == 'master' && needs.find-dbt-changes.outputs.changes == 'true'
     with:
       TARGET_COMPOSER_BUCKET: "europe-west1-data-composer--00137508-bucket"
       ENV_SHORT_NAME: "stg"
-    uses: ./.github/workflows/reusable_dbt_compile.yml
+    uses: ./.github/workflows/reusable_dbt_workflow.yml
```
Comment (lmontier-pass) [inline .github/workflows/CI_workflow.yml:39]: :+1:

File: README.md [hunk 4]
```diff
@@ -110,123 +110,7 @@ Les dags sont déployés automatiquement lors d'un merge sur master / production
...
-    C --> E
-    D --> E
-    E -->|merge master| H[Déploiement de Composer & DBT en Staging]
-    E -->|merge production| F[Déploiement de Composer & DBT en Dev]
-    F --> G[Déploiement de Composer & DBT en Production]
-    
-
-```
-
+Pipelines détaillées dans .github/workflows/README.md
```
Comment (lmontier-pass) [inline README.md:113]: Tu peux créer un lien markdow 
`Pipelines détaillées dans .github/workflows/README.md`
-> 
`Pipelines détaillées dans le [README de github Action](.github/workflows/README.md)`

**Review by lmontier-pass — DISMISSED** [review]

File: .github/workflows/CD_workflow.yml (hunk 1 above)
Comment (valoumiaou) [reply .github/workflows/CD_workflow.yml:1]: Le name c'est pour l'affichage dans Actions, Je me. dis qu'on pouvais avoir un truc plus clean que les noms de files.
D'autant que la confusion est quasi impossible, ca se déclenche sur des events différents et le mapping:
"Integration workflow" -> CI_workflow.yml
"Deployement workflow" -> CD_workflow.yml
est quand meme vachement transparent. apres j'ai pas mis de billets dans ce naming donc comme vous preferez

**Review by lmontier-pass — APPROVED** [review]

**Pull request merged**